packages/codegen-ui-react/lib/utils/forms/array-field-component.ts

```diff
@@ -267,29 +267,7 @@ export const generateArrayFieldComponent = () => {
                 [
                   factory.createIfStatement(
                     factory.createBinaryExpression(
-                      factory.createParenthesizedExpression(
-                        factory.createBinaryExpression(
```

Comment:
This condition - `(currentFieldValue !== undefined || currentFieldValue !== '')` is a bug. It will not rule out any nullish value :)

Reply:
I believe this was is verbose like this because we didn't want it to apply to `0`, `false`, etc.

Reply:
Then should use `&`.